File: SlimNotes CSharp/Mainnote.cs
using System;
using System.ComponentModel;
using System.Diagnostics;
using System.Drawing;
using System.Media;
using System.Threading;
using System.Windows.Forms;
//remove unused namespaces when done

namespace SlimNotes_CSharp
{
    public partial class Mainnote : Form
    {
        public Mainnote()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            //this thing doesn't work, further study is needed
            Trace.TraceInformation("hi mom im on tv");
            Console.WriteLine("Form loaded.");
        }

        private void ClearConsoleOutputToolStripMenuItem_Click(object sender, EventArgs e)
        {
            Console.Clear();
        }

        private void ContextMenuStrip1_Opening(object sender, CancelEventArgs e)
        {
            //sound that plays when the menu is opened
            SoundPlayer audio = new SoundPlayer(SlimNotes_CSharp.Properties.Resources.boopsfx1);
            audio.Play();
            Console.WriteLine("boopsfx1 played.");
        }

        private void CutToolStripMenuItem_Click(object sender, EventArgs e)
        {
            richTextBox1.Cut();
            Console.WriteLine("Cut.");
        }

        private void CopyToolStripMenuItem_Click(object sender, EventArgs e)
        {
            richTextBox1.Copy();
            Console.WriteLine("Copy.");
        }

        private void PasteToolStripMenuItem_Click(object sender, EventArgs e)
        {
            richTextBox1.Paste();
            Console.WriteLine("Paste.");
        }

        private void YellowDefaultToolStripMenuItem_Click(object sender, EventArgs e)
        {
            richTextBox1.BackColor = Color.Yellow;
            Console.WriteLine("Note color changed to yellow.");
        }

        private void OrangeToolStripMenuItem_Click(object sender, EventArgs e)
        {
            richTextBox1.BackColor = Color.Orange;
            Console.WriteLine("Note color changed to orange.");
        }

        private void BlueToolStripMenuItem_Click(object sender, EventArgs e)
        {
            richTextBox1.BackColor = Color.LightBlue;
            Console.WriteLine("Note color changed to blue.");
        }

        private void WhiteToolStripMenuItem_Click(object sender, EventArgs e)
        {
            richTextBox1.BackColor = Color.White;
            Console.WriteLine("Note color changed to white.");
        }

        private void CustomToolStripMenuItem_Click(object sender, EventArgs e)
        {
            if (colorDialog1.ShowDialog() == DialogResult.OK)
                richTextBox1.BackColor = colorDialog1.Color;
            Console.WriteLine("Note color changed to a custom color.");
        }

        private void FontsToolStripMenuItem_Click(object sender, EventArgs e)
        {
            //opens the font change dialog and sets it
            fontDialog1.ShowColor = true;
            fontDialog1.Font = richTextBox1.Font;
            fontDialog1.Color = richTextBox1.ForeColor;
            if (fontDialog1.ShowDialog() != DialogResult.Cancel)
            {
                richTextBox1.Font = fontDialog1.Font;
                richTextBox1.ForeColor = fontDialog1.Color;
                Console.WriteLine("Font changed.");
            }
        }

        private void NewNoteToolStripMenuItem_Click(object sender, EventArgs e)
        {
            //poor way of handling the creation of a new note because if the first note is closed, ALL OF THEM CLOSE
            new Thread(() => new Mainnote().ShowDialog()).Start();
            Console.WriteLine("New note window created.");
            //update: (4/22/21) finally fixed :DDDDD
         }

        private void OpenToolStripMenuItem_Click(object sender, EventArgs e)
        {

            {
                MessageBox.Show("This feature is not availible in this unfinished version of SlimNotes.", "Alert");  
                //i hate my life
                //var fileContent = string.Empty;
                //var filePath = string.Empty;
                //if (openFileDialog1.ShowDialog() == DialogResult.OK)
                //{
                //    richTextBox1.Text = fileContent;
                    
                //}
                
            }
        }

        private void saveAsToolStripMenuItem_Click(object sender, EventArgs e)
        {
            //imagine having enough of a brain to properly port this code from the old VB .NET version
            MessageBox.Show("This feature is not availible in this unfinished version of SlimNotes.", "Alert");
        }

        private void exitToolStripMenuItem_Click(object sender, EventArgs e)
        {
            if (MessageBox.Show("Are you sure you want to exit? This will close all windows.", "Alert", MessageBoxButtons.YesNo) == DialogResult.Yes)
            {
                Application.Exit();
            }
        }
    }
}

File: working title/mainnote.vb
Public Class Mainnote
    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        'CRAZY HAMBORGER
        'steven esso is a simp
        Console.WriteLine("Form loaded.")
    End Sub
    Private Sub Form1_Resize(sender As Object, e As EventArgs) Handles Me.Resize
        'notearea.Size = Me.Size
        'Console.WriteLine("Sized RTB to window.")
        'Me.KeyPreview = True
    End Sub
    Private Sub CutToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles CutToolStripMenuItem.Click
        notearea.Cut()
        Console.WriteLine("Cut.")
    End Sub
    Private Sub CopyToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles CopyToolStripMenuItem.Click
        notearea.Copy()
        Console.WriteLine("Copy.")
    End Sub
    Private Sub PasteToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles PasteToolStripMenuItem.Click
        notearea.Paste()
        Console.WriteLine("Paste.")
    End Sub
    Private Sub ContextMenuStrip1_Opening(ByVal sender As Object, ByVal e As System.ComponentModel.CancelEventArgs) Handles ContextMenuStrip1.Opening
        My.Computer.Audio.Play(My.Resources.boopsfx1, AudioPlayMode.Background)
        Console.WriteLine("boopsfx1 played.")
    End Sub
    Private Sub SaveAsToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles SaveAsToolStripMenuItem.Click
        SaveFileDialog1.Filter = "SlimNote Files (*.snf*)|*.snf"
        If SaveFileDialog1.ShowDialog = Windows.Forms.DialogResult.OK _
         Then
            My.Computer.FileSystem.WriteAllText _
            (SaveFileDialog1.FileName, notearea.Text, True)

        End If
    End Sub
    Private Sub SaveFileDialog1_FileOk(sender As Object, e As System.ComponentModel.CancelEventArgs) Handles SaveFileDialog1.FileOk
        MsgBox("Saved.")
    End Sub
    Private Sub OpenToolStripMenuItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles OpenToolStripMenuItem.Click
        'add confirmation later
        OpenFileDialog1.ShowDialog()
        notearea.Text = My.Computer.FileSystem.ReadAllText(OpenFileDialog1.FileName)
        Text = OpenFileDialog1.FileName
        Console.WriteLine("File opened.")
    End Sub
    Private Sub WhiteDefaultToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles WhiteDefaultToolStripMenuItem.Click
        notearea.BackColor = Color.White
        Console.WriteLine("Note color changed to white.")
    End Sub
    Private Sub YellowToolStripMenuItem1_Click(sender As Object, e As EventArgs) Handles YellowToolStripMenuItem1.Click
        notearea.BackColor = Color.Yellow
        Console.WriteLine("Note color changed to yellow.")
    End Sub
    Private Sub OrangeToolStripMenuItem1_Click(sender As Object, e As EventArgs) Handles OrangeToolStripMenuItem1.Click
        notearea.BackColor = Color.Orange
        Console.WriteLine("Note color changed to orange.")
    End Sub
    Private Sub BlueToolStripMenuItem1_Click(sender As Object, e As EventArgs) Handles BlueToolStripMenuItem1.Click
        notearea.BackColor = Color.LightBlue
        Console.WriteLine("Note color changed to blue.")
    End Sub
    Private Sub FontsWIPToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles FontsWIPToolStripMenuItem.Click
        If FontDialog1.ShowDialog <> Windows.Forms.DialogResult.Cancel Then
            notearea.ForeColor = FontDialog1.Color
            notearea.Font = FontDialog1.Font
            Console.WriteLine("Font changed.")
        End If
    End Sub
    Private Sub CustomToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles CustomToolStripMenuItem.Click
        If ColorDialog1.ShowDialog <> DialogResult.Cancel Then
            notearea.BackColor = ColorDialog1.Color
            Console.WriteLine("Note color changed to a custom color.")
        End If
    End Sub
    Private Sub ThisWindowToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles ThisWindowToolStripMenuItem.Click
        'might scrap code here
        If notearea.Modified Then
        End If
        Dim M As MsgBoxResult
        M = MsgBox("Save changes?", MsgBoxStyle.YesNoCancel)
        If M = MsgBoxResult.Cancel Then
        ElseIf M = MsgBoxResult.No Then
            notearea.Clear()
        ElseIf M = MsgBoxResult.Yes Then
            SaveFileDialog1.Filter = "SlimNote Files (*.snf*)|*.snf"
            If SaveFileDialog1.ShowDialog = Windows.Forms.DialogResult.OK _
             Then
                My.Computer.FileSystem.WriteAllText _
                (SaveFileDialog1.FileName, notearea.Text, True)
                notearea.Clear()
                Console.WriteLine("New note created in same window.")
            End If
        End If
    End Sub
    Private Sub NewWindowToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles NewWindowToolStripMenuItem.Click
        Shell("SlimNotes.exe")
    End Sub
    Private Sub ClearConsoleOutputToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles ClearConsoleOutputToolStripMenuItem.Click
        Console.Clear()
    End Sub
End Class